### DISCUSSION TITLE
Reuse of .deps.json for dependency loading

### DISCUSSION THREAD

**Discussion by MichalMKsiazek** [pythonnet/pythonnet#2414] [Q&A] [answered]
I'm working on a project at work where I am attempting to reuse an internal tool develop in .Net6 in python. I am having some trouble managing all the dependencies and was curious if there is a function in Python.NET that allows you use load the dependencies using .deps.json?

**Comment by MichalMKsiazek** [comment, marked as answer]
Realised this is not needed

**Reply by niander** [reply]
Care to elaborate why it was not needed for your case? I am trying to load a library in a multi-platform manner. After inspecting the assembly logs with "dotnet-trace" it doesn't seem that the dependencies are loaded in a manner that the runtime would do. Wrong versions of the DLL are being loaded. I am unsure why this is happening
